src/helpers/create/create-microservice.test.js

```diff
@@ -0,0 +1,152 @@
+import { config } from '~/src/config'
+import { MongoClient } from 'mongodb'
+import { fetchTeam } from '~/src/helpers/fetch-team'
+import { createMicroservice } from '~/src/helpers/create/create-microservice'
+import { statuses } from '~/src/constants/statuses'
+import { triggerWorkflow } from '~/src/helpers/create/workflows/trigger-workflow'
+
+jest.mock('~/src/helpers/fetch-team', () => ({
+  fetchTeam: jest.fn()
+}))
+
+jest.mock('~/src/helpers/create/workflows/trigger-workflow', () => ({
+  triggerWorkflow: jest.fn()
+}))
+
+const logger = { info: jest.fn(), error: jest.fn(), warn: jest.fn() }
+let connection
+let db
+const service = 'test-service-12345'
+
+beforeEach(async () => {
+  await db.collection('status').deleteMany({ repositoryName: service })
+})
+
+beforeAll(async () => {
+  connection = await MongoClient.connect(globalThis.__MONGO_URI__, {
```

Comment:
This doesn't feel right. @chotai mentioned this earlier. I'll have a look at his branch shortly